test/test_jit.py

```diff
@@ -11921,7 +11923,8 @@ def cell(x, hx, cx, w_ih, w_hh, b_ih, b_hh):
     def test_lstm_traced_cuda(self):
         inputs = get_lstm_inputs('cuda')
         ge = self.checkTrace(LSTMCellF, inputs)
-        self.assertExpectedGraph(ge.graph_for(*inputs))
+        graph = ge.graph_for(*inputs)
+        FileCheck().check("FusionGroup").run(str(graph))
```

Comment:
Would be good to check:
- there is exactly one fusion group
- there are no chunks outside the fusion group
- there are no fusible operations outside the fusion group

Reply:
what do you mean exactly by there is no fusible operations outside the fusion group?

Reply:
For LSTM forward, check that there aren't add / sigmoid / tanh operations that are outside the fusion group